pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -788,3 +791,42 @@ def sort_legend_elem(elem):
         value = -elem.get(geom_type, 0)
 
         return category, value
+
+    def get_custom_wms_params(self, params):
+        """
+        From a given dictionary filter out all the parameters that are specified in the config
+        and return these.
+        Only values which exist in the given params are added.
+        If there is nothing configured or the config is not a list
+        the configured wms_url_params are used as fallback.
+
+        Args:
+            params (dict): Parameters available in the URL
+
+        Returns:
+            custom_params (dict): dictionary of filtered params
+        """
+        wms_url_params = Config.get('print', {}).get('wms_url_keep_params', False)
+        if not isinstance(wms_url_params, list):
+            # If no config exists or config is not as expected fall back to use standard wms_url_params
+            return self.get_wms_url_params()
+
+        custom_params = {
+            wms_keys: self.string_check(wms_values)
+            for wms_keys, wms_values in params.items()
+            if wms_keys in wms_url_params
+        }
+        return custom_params
+
+    @staticmethod
+    def string_check(value):
+        """
+        Check if the value given is a string. It it is not jon the values using comma separator.
```

Comment:
Typos in the second sentence:
If it is not, assume it is a list and join the values using comma separator.